plugins/generator-1.21.1/neoforge-1.21.1/procedures/gui_set_items.java.ftl

```diff
@@ -1,7 +1,8 @@
 <#include "mcitems.ftl">
-if(${input$entity} instanceof Player _player && _player.containerMenu instanceof ${JavaModName}Menus.MenuAccessor _menu) {
+<@head>if (${input$entity} instanceof Player _player && _player.containerMenu instanceof ${JavaModName}Menus.MenuAccessor _menu) {</@head>
 	ItemStack _setstack = ${mappedMCItemToItemStackCode(input$item, 1)}.copy();

```

Comment:
This definitely lacks cbi, if multiple of same type, it will create same variable. Check other templates too